components/script/dom/webgl_extensions/extensions.rs

```diff
@@ -52,7 +64,9 @@ impl Default for WebGLExtensionFeatures {
             disabled_tex_types: DEFAULT_DISABLED_TEX_TYPES.iter().cloned().collect(),
             not_filterable_tex_types: DEFAULT_NOT_FILTERABLE_TEX_TYPES.iter().cloned().collect(),
             effective_tex_internal_formats: HashMap::new(),
-            query_parameter_handlers: HashMap::new()
+            query_parameter_handlers: HashMap::new(),
+            hint_targets: HashSet::new(),
+            disabled_get_parameter_names: DEFAULT_DISABLED_GET_PARAMETER_NAMES.iter().cloned().collect(),
```

Comment:
Sure, I mean, for the purpose of looking up into the `HashSet`, probably it's slower (as it stands, and probably until it reaches ~20 different values) than a simple array.